src/Server.ts

```diff
@@ -1,5 +1,8 @@
-// create a class and define methods according to the ticket#39521
-import * as express from "express";
+// create a class and define methods according to the ticket#39522
+import * as express from 'express';
+import * as bodyParser from 'body-parser';
+import { notFoundHandler, errorHandler } from './libs/routes';
+import notFoundRoute from "./libs/routes/notFoundRoute";
 class Server {
```

Comment:
initBodyParser implementation is missing
notFoundRoute and errorHandler routes are also missing.

Reply:
done